docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -136,10 +136,10 @@ connector.
       - Description
       - Default
     * - ``delta.domain-compaction-threshold``
-      - Sets the number of transactions to act as threshold. Once reached the
-        connector initiates compaction of the underlying files and the delta
-        files. A higher compaction threshold means reading less data from the
-        underlying data source, but a higher memory and network consumption.
+      - Sets the number of transactions to act as a threshold. After reaching
+        the threshold, the connector initiates compacting a large IN or OR
+        clause into a min-max range predicate for pushdown into an ORC or
+        Parquet reader.
```

Comment:
I think we should simplify this down, there's a lot of components that are transparent to the user. If the point of the threshold is to begin pushing down clauses to Delta Lake after the transaction threshold is met, which is what this reads like to me, then this can be rewritten to "After reaching the transaction threshold, the connector compacts its workload by pushing down large `IN` or `OR` clauses to Delta Lake."

Would double-check that language in case I'm misunderstanding.